app/services/utm_service.rb

```diff
@@ -9,11 +9,17 @@ class UTMService
     sign_in: "sign-in",
     challenge_partnership: "challenge-partnership",
     partnership_notification: "partnership-notification",
+    june_private_beta: "june-private-beta",
   }.freeze
 
-  def self.email(campaign)
+  SOURCES = {
+    service: "cpdservice",
+    private_beta: "cpdprivatebeta",
```

Comment:
Do we expect a bunch of other emails / links and want to link them together under the private beta umbrella?

If not, is that `cpdprivatebeta` a bit of an overkill?

Reply:
It was requested 🤷

Reply:
https://ukgovernmentdfe.slack.com/archives/C01J1J4FM50/p1623761868488000